### DISCUSSION TITLE
301 redirect in vercel

### DISCUSSION THREAD

**Discussion by ashurajchandra** [vercel/vercel#6687] [Help] [answered]
i am using gatsby for my website and hosted it on vercel and now i want to make 301 redirect in vercel but i have no idea how to do it please help me get better understanding of this process

**Comment by leerob** [comment, marked as answer]
You can do some redirects inside the Vercel UI: https://vercel.com/docs/projects/custom-domains#redirecting-domains

Or, you can programmatically use **vercel.json** like so...

```json
{
  "redirects": [
    { "source": "/old", "destination": "/new" },
  ]
}
```

The default is a permanent redirect 👍 https://vercel.com/docs/cli#project-configuration/redirects
